File: MACHINE RUN 11-11-2020/check greatest of 3 numbers/greater of 3 numbers.cpp
#include<iostream>
using namespace std;
class Great_Ex
{
    int num1,num2,num3;
    public:
        void Check_greater()
        {
            cout<<"Enter three numbers";
            cin>>num1>>num2>>num3;
            if(num1>num2 && num1>num3)
            {
                cout<<"num1 is greater";
            }
        else if(num2>num1 && num2>num3)
            {
                cout<<"num2 is greater";

            }
            else{
                cout<<"num3 is greater";
            }
        }
};

int main()
{
    Great_Ex g1;
    g1.Check_greater();
}

File: ASSIGNMENT 2(Java)/greaterof3numbers.java
import java.util.Scanner;
class Greater
{
    public static void main(String args[])
    {
        int num1,num2,num3;
        System.out.println("Enter First Number : ");
        Scanner sc = new Scanner(System.in);
        num1 = sc.nextInt();
        System.out.println("Enter Second Number : ");
        num2 = sc.nextInt();
        System.out.println("Enter Third Number : ");
        num3 = sc.nextInt();	
if(num1>num2 && num1>num3)
        {
            System.out.println("The Number1 is Greater");
        }
      else if(num2>num1 && num2>num3)
        {
            System.out.println("The Number2 is Greater");
        }
else
{
   System.out.println("The Number3 is Greater");
}
    }
}
 